aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -1543,6 +1559,50 @@ def get_openapi_schema(
 
         return OpenAPI(**output)
 
+    @staticmethod
+    def _get_openapi_servers(servers: Optional[List["Server"]]) -> List["Server"]:
+        from aws_lambda_powertools.event_handler.openapi.models import Server
+
+        # If the 'servers' property is not provided or is an empty array,
+        # the default behavior is to return a Server Object with a URL value of "/".
+        return servers if servers else [Server(url="/")]
+
+    @staticmethod
+    def _get_openapi_security(
+        security: Optional[List[Dict[str, List[str]]]],
+        security_schemes: Optional[Dict[str, "SecurityScheme"]],
+    ) -> Optional[List[Dict[str, List[str]]]]:
+        if security:
+            if not security_schemes:
+                raise ValueError("security_schemes must be provided if security is provided")
+
+            # Check if all keys in security are present in the security_schemes
+            if not all(key in security_schemes for sec in security for key in sec):
+                raise ValueError("Some security schemes not found in security_schemes")
+
+            return security
+        else:
+            return None
```

Comment:
Do you think we can simplify this function by returning None in advance if the security is not present?

Something like this
```suggestion
    @staticmethod
    def _get_openapi_security(
        security: Optional[List[Dict[str, List[str]]]],
        security_schemes: Optional[Dict[str, "SecurityScheme"]],
    ) -> Optional[List[Dict[str, List[str]]]]:
        if not security:
            return None

        if not security_schemes:
            raise ValueError("security_schemes must be provided if security is provided")

        # Check if all keys in security are present in the security_schemes
        if any(key not in security_schemes for sec in security for key in sec):
            raise ValueError("Some security schemes not found in security_schemes")

        return security
```

Reply:
This is 1 million times better, thank you!